### PR TITLE
Add fastapi-serving one card

### PR DESCRIPTION
**Pull request by hzjane**
## Description
Add fastapi-serving one card

simple performance vs fastchat：
llama2-7b1024-128 fp8
fastchat: 436.6ms - 23.2ms
lightweight-serving:  378.9ms - 21.8ms

### 4. How to test?
- [ ] N/A
- [x] Unit test:  pr test passed here: https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/9949404410
- [x] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/example/GPU/Pipeline-Parallel-FastAPI/serving.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from transformers import AutoTokenizer
+import uvicorn
+import asyncio
+import argparse
+from ipex_llm.serving.api import FastApp
+from ipex_llm.serving.api import ModelWorker
+logger = logging.get_logger(__name__)
+
+async def main():
+    parser = argparse.ArgumentParser(description='Predict Tokens using fastapi by leveraging DeepSpeed-AutoTP')
```
Comment (xiangyuT) [inline python/llm/example/GPU/Pipeline-Parallel-FastAPI/serving.py:29]: leveraging DeepSpeed-AutoTP -> leveraging ipex-llm?

File: python/llm/src/ipex_llm/serving/api/model_worker.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+import torch
+from transformers.utils import logging
+import time
+import asyncio
+from transformers import TextIteratorStreamer
+logger = logging.get_logger(__name__)
+
+
+class ModelWorker:
```
Comment (xiangyuT) [inline python/llm/src/ipex_llm/serving/api/model_worker.py:25]: Rename `ModelWorker` and `ModelRunner` to `ModelWorker` and `PPModelWorker`？The current name may be kind of confusing

File: python/llm/example/GPU/Pipeline-Parallel-FastAPI/README.md [hunk 3]
```diff
@@ -50,7 +50,14 @@ pip install transformers==4.40.0
 pip install trl==0.8.1
```
Comment (glorysdj) [inline python/llm/example/GPU/Pipeline-Parallel-FastAPI/README.md:50]: rename this example to Pipeline-Parallel-Serving

File: python/llm/src/ipex_llm/serving/api/api_server.py [hunk 4]
```diff
@@ -0,0 +1,315 @@
+#
```
Comment (glorysdj) [inline python/llm/src/ipex_llm/serving/api/api_server.py:1]: rename api to fastapi

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**